### PR TITLE
Add visual styles to search

### PR DESCRIPTION
**Pull request by cshinn**
Visual updates to landing page search bar. Does not include css to make the search bar sticky.

screen shot 2017-04-05 at 2 52 16 pm

### CODE REVIEW

File: src/styles/landing-page.less [hunk 1]
```diff
@@ -11,6 +10,14 @@
   }
 }
 
+.landing-search-area {
+  border-bottom: solid 1px @color-pf-blue-200;
+  padding: (@grid-gutter-width / 4);
+  @media (min-width: @screen-sm-min) {
+      padding: (@grid-gutter-width / 3);
```
Comment (spadgett) [inline src/styles/landing-page.less]: nit: indentation

File: src/styles/landing-page.less [hunk 2]
```diff
@@ -19,39 +26,48 @@
...
   }
   .dropdown-menu {
+    margin-top: 0;
     width: 100%;
     // Styles need to be specific enough to override defaults.
     > li > a.catalog-search-match {
       padding-bottom: 5px;
       padding-top: 5px;
     }
+    // important necessary to override defaults
```
Comment (spadgett) [inline src/styles/landing-page.less]: I'd be more specific: "important needed to override styles declared important in Patternfly"

File: dist/origin-web-catalogs.css [hunk 3]
```diff
@@ -34,9 +34,6 @@
     opacity: 0;
   }
 }
-.landing-main-area {
-  padding: 20px;
```
Comment (spadgett) [inline dist/origin-web-catalogs.css]: Wonder if we shouldn't give a temporary margin to the headings if we're removing the padding on the entire page.

**Comment by spadgett** [comment]
I noticed that the individual tags highlight when you mouse over them. It makes it look like the tag does something special when clicked.

**Comment by cshinn** [comment]
@spadgett I've fixed the hover issue. It won't let me squash this commit for some reason though...

**Comment by spadgett** [comment]
We should add the hourglass icon to the search input, either in this PR or a follow-on

**Comment by spadgett** [comment]
@cshinn Conflict now that I merged #65 :/

**Comment by jeff-phillips-18** [comment]
I agree, the search component seems small to me.

**Comment by spadgett** [comment]
@chsinn is also looking at fixing the search to the top. Suggest we use this PR for styling the search result and a follow on for search input styling (position, size, and hourglass icon)

**Pull request merged**